src/cmd/acme/rows.c

```diff
@@ -433,10 +436,10 @@ rowdump(Row *row, char *file)
 			Bwrite(b, buf, strlen(buf));
 			m = min(RBUFSIZE, w->tag.file->b.nc);
 			bufread(&w->tag.file->b, 0, r, m);
-			n = 0;
-			while(n<m && r[n]!='\n')
-				n++;
-			Bprint(b, "%.*S\n", n, r);
+			for(n=0;n<m;n++)
```

Comment:
Please look at the style in the surrounding code. There should be spaces after the semicolons in the for and around the operator (==) in the if.